tests/app/main/views/test_dashboard.py

```diff
@@ -666,12 +666,12 @@ def test_should_show_redirect_from_template_history(
     )
 
 
-@freeze_time("2017-01-01 12:00")  # 4 months into 2016 financial year
+@freeze_time("2017-1-1 12:00")  # Switching to calendar year
```

Comment:
For the most part all of this work is based on fiscal year still, yes?  I see the date itself isn't actually different so it doesn't look like there's a real change here, but the comment caught my eye if the intent was to actually switch to calendar year here.

In this case I'd also recommend keeping the leading zeros as that is generally expected when dealing with date and datetime values in Python for days and months (e.g., like in [`freezegun`'s own docs](https://github.com/spulec/freezegun?tab=readme-ov-file#decorator)):

```suggestion
@freeze_time("2017-01-01 12:00")  # Switching to calendar year
```

Reply:
Yeah, we are still based on fiscal year from October to September. I probably got ahead of myself on this comment. Thank you for the explanation!

Reply:
No worries, and you're welcome!